apps/web/src/pages/EmployeeProfile/EmployeeProfilePage.tsx

```diff
@@ -39,15 +48,24 @@ const EmployeeProfile_Fragment = graphql(/** GraphQL */ `
     isWorkEmailVerified
     employeeProfile {
       ...EmployeeProfileGoalsWorkStyle
+      ...EmployeeProfileCareerDevelopment
     }
   }
 `);
 
 interface EmployeeProfileProps {
+  userId: string;
   userQuery: FragmentType<typeof EmployeeProfile_Fragment>;
+  careerDevelopmentOptionsQuery: FragmentType<
+    typeof EmployeeProfileCareerDevelopmentOptions_Fragment
+  >;
 }
 
-const EmployeeProfile = ({ userQuery }: EmployeeProfileProps) => {
+const EmployeeProfile = ({
+  userId,
+  userQuery,
+  careerDevelopmentOptionsQuery,
+}: EmployeeProfileProps) => {
```

Comment:
Its a small clean up. 
since userId , userQuery are retrieved from data , they can be assigned inside EmployeeProfile component. 
passing the data that you already have causes duplication. 
I meant this way , 

```
const EmployeeProfile = ({
careerDevelopmentOptionsQuery,
}: EmployeeProfileProps) => { 
const intl = useIntl();
 const paths = useRoutes();
 const userQuery = careerDevelopmentOptionsQuery.me;
 const userId = userQuery.id;
 
const user = getFragment(EmployeeProfile_Fragment, userQuery);
```
	
 ...

Reply:
I removed the userId, but for the other two props, I think I should leave them since they are both fragments.

Reply:
This makes sense. One is a fragment on User and one is a fragment on Query, right?

Reply:
I could pass down the entire query in the props then split it up there, but it makes it easier with the typing to do it like this.

Reply:
Nvm Brinda was right, I couldn't figure out this type error, but Brinda helped me see the light :bulb: 
Changed in [c0ec84c](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12680/commits/c0ec84c3f3e0afadb3e0556d80d7fd2811fa26af)